torch/csrc/distributed/c10d/init.cpp

```diff
@@ -223,6 +223,26 @@ PyObject* c10d_init(PyObject* _unused) {
               py::arg("output_tensor"),
               py::arg("tensors"),
               py::arg("root"),
+              py::call_guard<py::gil_scoped_release>())
+
+          .def(
+              "send",
+              &::c10d::ProcessGroup::send,
+              py::call_guard<py::gil_scoped_release>())
+
+          .def(
+              "recv",
+              &::c10d::ProcessGroup::recv,
+              py::call_guard<py::gil_scoped_release>())
+
+          .def(
+              "recvAnysource",
```

Comment:
It would be better to stick to our regular conventions for Python function names i.e. `recv_anysource` (or even better just make it an overload of `recv`).